deployment/helm/topology-aware/templates/daemonset.yaml

```diff
@@ -28,9 +28,16 @@ spec:
     {{- with .Values.nodeSelector }}
         {{- toYaml . | nindent 8 }}
     {{- end }}
-      {{- if .Values.nri.patchRuntimeConfig }}
+      {{- if .Values.nri.runtime.patchConfig }}
       initContainers:
       - name: patch-runtime
+        {{- if (not (or (eq .Values.nri.runtime.config nil) (eq .Values.nri.runtime.config.pluginRegistrationTimeout ""))) }}
+        args:
+          - -nri-plugin-registration-timeout
+          - {{ .Values.nri.runtime.config.pluginRegistrationTimeout }}
+          - -nri-plugin-request-timeout
+          - {{ .Values.nri.runtime.config.pluginRequestTimeout }}
```

Comment:
Hmm, if only one of those is defined we end up in trouble(?)

Reply:
Helm shouldn't accept that. The values JSON schema effectively allows either both or neither to be specified.

Reply:
K, then in the template it should use `and` instead of `or`(?)